components/Search/forms/SearchCasesForm.jsx

```diff
@@ -120,6 +122,26 @@ const SearchCasesForm = ({ onFormSubmit, defaultValues }) => {
       </div>
 
       <Button label="Search" type="submit" disabled={!isDirty} />
+      <span
+        className="govuk-body"
+        onClick={() => {
+          reset({
+            start_date: '',
```

Comment:
you need to set it to `null` otherwise in the next search it will complain that is required